### DISCUSSION TITLE
Permissions / release of imagery for use in articles, blog posts, papers, case studies, etc..

### DISCUSSION THREAD

**Discussion by wesley-dean-flexion** [oxsecurity/megalinter#3443] [General]
Is there a process for obtaining permissions or a release to use imagery from the Megalinter.io site in publications like articles, blog posts, papers, etc.?

**Background**
I'm writing an article about Megalinter and how awesome it is.  My editor and illustrator really like some of the images on the megalinter.io site.  They **love** the [llama](https://github.com/oxsecurity/megalinter/blob/main/docs/assets/images/mega-linter-logo.png).

However, they're concerned about potential copyright issues.  While I get that Megalinter is an open-source project, I would like to be cautious about using imagery, especially with the logo given the presence of the ox (i.e., mascot for Ox Security).

**Comment by nvuillam** [comment]
@wesley-dean-flexion you can use any image that you can find in the repo , while it is clearly associated to MegaLinter :)

Thanks for writing an article, I'll add it in the documentation 😍

**Reply by wesley-dean-flexion** [reply]
https://flexion.us/blog/5-ways-megalinter-upped-our-devsecops-game/

I submitted #3596 a few minutes ago to add it to the list of English articles

**Reply by nvuillam** [reply]
I'll read it and validate the PR ASAP, many thanks <3

**Reply by wesley-dean-flexion** [reply]
I've 3 more articles in the pipeline.  Given that it takes a few months for them to work their way through, they probably won't surface until late summer or early fall.

**Reply by nvuillam** [reply]
@wesley-dean-flexion I might give you new material for new articles... :p 

SPOILERS

![image](https://github.com/oxsecurity/megalinter/assets/17500430/b1ccfc1b-7ff1-46a3-8ad5-6164c2f741f0)

![image](https://github.com/oxsecurity/megalinter/assets/17500430/ed85aca9-da6c-4743-bb39-047605087aa5)

**Reply by wesley-dean-flexion** [reply]
:open_mouth: Grafana!  Very cool!!!

Sorry to keep editing this (better than spamming with lots of notifications, right?).

What a great idea. Such a great idea.  My mind is racing with places where this'll be applicable.

**Comment by wesley-dean-flexion** [comment]
@nvuillam the article is due out 9 May.  I'm already working on two follow-up articles.  I'll happily provide links as they become available.  Thank you for your support!  :smile:

**Reply by nvuillam** [reply]
Such links will end here for sure :) https://megalinter.io/latest/articles/#english-articles

**Reply by wesley-dean-flexion** [reply]
I hope so.  Mon Francais est tres mauvais.
